src/firmware_new/boards/frankie_v1/io/infineon_TLE5009_E1000.h

```diff
@@ -0,0 +1,66 @@
+#pragma once
+
+#include <math.h>
+/**
+ * This file is an abstraction around the Infineon TLE5009 E1000 GMR angle sensor
+ *
+ * Datasheet:
+ * https://www.infineon.com/dgdl/Infineon-TLE5009_FDS-DataSheet-v01_01-en.pdf?fileId=db3a304330f686060131421d8ddd56b0
+ */
+
+typedef struct InfineonTLE5009E1000 InfineonTLE5009E1000_t;
+
+/**
+ * Creates an infineon TLE5009 E1000 abstraction given the calibration parameters
+ *
+ * @param x_max_value [in] The maximum value observed over a full period of the X output
+ * waveform
+ *
+ * @param x_min_value [in] The minimum value observed over a full period of the X output
+ * waveform
+ *
+ * @param y_max_value [in] The maximum value observed over a full period of the Y output
+ * waveform
+ *
+ * @param y_min_value [in] The minimum value observed over a full period of the Y output
+ * waveform
+ *
+ * @param x_magnitude_45_degrees [in] The X waveform magnitude at 45 degrees (as read by
+ * the sensor)
+ *
+ * @param y_magnitude_45_degrees [in] The Y waveform magnitude at 45 degrees (as read by
+ * the sensor)
+ *
+ * @param x_magnitude_135_degrees The X waveform magnitude at 135 degrees (as read by the
+ * sensor)
+ *
+ * @param y_magnitude_135_degrees [in] The Y waveform magnitude at 135 degrees (as read by
+ * the sensor)
+ *
+ * @return Infineon TLE5009E1000 abstraction
+ */
+InfineonTLE5009E1000_t* io_infineon_TLE5009_E1000_create(
+    float x_max_value, float x_min_value, float y_max_value, float y_min_value,
+    float x_magnitude_45_degrees, float y_magnitude_45_degrees,
+    float x_magnitude_135_degrees, float y_magnitude_135_degrees);
+
+/**
+ * Destroys infineon sensor abstraction
```

Comment:
Nit: Keep it consistent and capitalize `Infineon`.

Reply:
Capatalized all non-function uses 👍